### PR TITLE
Remove macOS group constraint and bump some dep versions

### PR DESCRIPTION
**Pull request by Brantone**
_just an idea_

### CODE REVIEW

**Comment by rjaros87** [comment]
Please add to .rubocop.yml
```
Metrics/BlockLength:
  Max: 30
```
and sort gems in Gemfile
```
Offenses:

Gemfile:5:1: C: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem chefspec should appear before tailor.

gem 'chefspec', '~> 4.7'

^^^^^^^^^^^^^^^^^^^^^^^^

Gemfile:7:1: C: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem chef should appear before foodcritic.

gem 'chef', '~> 12.11'

^^^^^^^^^^^^^^^^^^^^^^

Gemfile:9:1: C: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem berkshelf should appear before rubocop.

gem 'berkshelf', '~> 4.3'

^^^^^^^^^^^^^^^^^^^^^^^^^

Gemfile:13:3: C: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem kitchen-vagrant should appear before test-kitchen.

  gem 'kitchen-vagrant', '~> 0.20.0'

  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Thanks :)

**Comment by rjaros87** [comment]
See mac os logs https://travis-ci.org/rjaros87/chef-android-sdk/jobs/195140524#L396 ` STDERR: chown: root: illegal group name`

In this situation you should create new dna.json with properly attributes and call properly bash command in https://github.com/rjaros87/chef-android-sdk/blob/windows/.travis.yml#L17 depends on platform (linux/mac). Can you do that? :)

**Comment by Brantone** [comment]
Yup, can touch that up. Might take a day or two (hurray for work deadlines).

**Comment by Brantone** [comment]
Note. I recognize the badness in not supporting Windows for the `packages` option. I should probably split it to separate branch if it helps.

File: recipes/unix.rb [hunk 1]
```diff
@@ -160,6 +161,30 @@
...
+    script "Install Android package #{package}" do
+      interpreter 'expect'
+      environment 'ANDROID_HOME' => android_home
+      path [File.join(android_home, 'tools/bin')]
+      user node['android-sdk']['owner']
+      group node['android-sdk']['group']
+      # TODO: use --force or not?
+      code <<-EOF
+          spawn #{sdkmanager_bin} "#{package}"
+          set timeout 1800
```
Comment (Brantone) [inline recipes/unix.rb]: Just realized that 1800 seconds seems ridiculous for a timeout.

**Comment by rjaros87** [comment]
Maybe add `sdkmanager` as a separate PR, because PR #33 on the original repo is growing :), ok?

**Comment by Brantone** [comment]
👍  sounds good.

**Comment by rjaros87** [comment]
Thank you for refactoring ;)

**Pull request merged**